### PR TITLE
refactor: Drop IE11 support

### PR DESCRIPTION
**Pull request by HenadzV**
Description:

Remove the full version of the polyfill for ESL UI site renderer Remove shims and targets =ES5 in the site codebase Remove ES5-related OOTB ESL polyfills except es5-target-shim.ts Drop support for IE11 and ES5 target

BREAKING CHANGES:

ESL UI site renderer and ESL polyfills no longer support IE11 and ES5 target.

Closes: 

Relates to:

### CODE REVIEW

**Comment by HenadzV** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/BROWSER_SUPPORT.md [hunk 1]
```diff
@@ -32,23 +26,6 @@ Exadel Smart Library does not have dependencies but uses the following list of n
 
 All of them are fully supported by modern browsers such as Chrome, Firefox, Safari or Edge (>43).
 
-In order to make ESL work in older browsers, you can use a "light" polyfills list of IntersectionObserver, ResizeObserver and Custom Elements (for older versions of Edge and Safari).
-
-Or make the library work in IE11 or Edge (<14) by using the "full" polyfills list provided.
+In order to make ESL work in older browsers, you can use a "light" polyfills list (for older versions Safari).
 
 See more details on what polyfill approach might look like in the demo pages source code.
```
Comment (abarmina) [inline docs/BROWSER_SUPPORT.md:31]:
```suggestion
```

File: docs/BROWSER_SUPPORT.md [hunk 2]
```diff
@@ -32,23 +26,6 @@ Exadel Smart Library does not have dependencies but uses the following list of n
 
 All of them are fully supported by modern browsers such as Chrome, Firefox, Safari or Edge (>43).
 
-In order to make ESL work in older browsers, you can use a "light" polyfills list of IntersectionObserver, ResizeObserver and Custom Elements (for older versions of Edge and Safari).
-
-Or make the library work in IE11 or Edge (<14) by using the "full" polyfills list provided.
+In order to make ESL work in older browsers, you can use a "light" polyfills list (for older versions Safari).
```
Comment (abarmina) [inline docs/BROWSER_SUPPORT.md:29]:
```suggestion
In order to make ESL work in older browsers, you can use a "light" polyfills list (for older versions of Safari).
```

**Review by abarmina — APPROVED** [review]

File: pages/webpack.config.js [hunk 3]
```diff
@@ -5,8 +5,6 @@ module.exports = {
   devtool: 'source-map',
   entry: {
     'localdev': './src/localdev.ts',
-    'polyfill-full': './src/polyfill-full.ts',
-    'polyfill-medium': './src/polyfill-medium.ts',
     'polyfill-light': './src/polyfill-light.ts'
```
Comment (dshovchko) [inline pages/webpack.config.js:8]: Do we need to keep the polyfill bundle suffix?

**Review by abarmina — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by ala-n — APPROVED** [review]

**Pull request merged**